jannovar-core/src/test/java/de/charite/compbio/jannovar/reference/AminoAcidChangeTest.java

```diff
@@ -20,49 +20,77 @@ public void setUp() throws Exception {
 	@Test
 	public void testShiftNoRefChange() {
 		AminoAcidChange origChange = new AminoAcidChange(3, "CC", "");
-		AminoAcidChange modChange = AminoAcidChangeNormalizer.normalizeDeletion(ref, "LLLCLLLCCC", origChange);
+		AminoAcidChange modChange = AminoAcidChangeNormalizer.shiftSynonymousChange(origChange, ref, "LLLCLLLCCC");
 		Assert.assertEquals(new AminoAcidChange(4, "CC", ""), modChange);
 	}
 
 	@Test
 	public void testShiftNoRefChangeEnd() {
 		AminoAcidChange origChange = new AminoAcidChange(9, "CC", "");
-		AminoAcidChange modChange = AminoAcidChangeNormalizer.normalizeDeletion(ref, "LLLCCCLLLC", origChange);
+		AminoAcidChange modChange = AminoAcidChangeNormalizer.shiftSynonymousChange(origChange, ref, "LLLCCCLLLC");
 		Assert.assertEquals(new AminoAcidChange(10, "CC", ""), modChange);
 	}
 
 	@Test
 	public void testShiftRefChange() {
 		AminoAcidChange origChange = new AminoAcidChange(3, "CLT", "");
-		AminoAcidChange modChange = AminoAcidChangeNormalizer.normalizeDeletion(ref2, "CLTCTTCL", origChange);
+		AminoAcidChange modChange = AminoAcidChangeNormalizer.shiftSynonymousChange(origChange, ref2, "CLTCTTCL");
 		Assert.assertEquals(new AminoAcidChange(4, "LTC", ""), modChange);
 	}
 
 	@Test
-	public void testShiftFrameshiftNoInsAA() {
+	public void testShiftDeletionFrameshiftNoInsAA() {
 		AminoAcidChange origChange = new AminoAcidChange(3, "CQY", "");
-		AminoAcidChange modChange = AminoAcidChangeNormalizer.normalizeDeletion("LLLCQYCLLL", "LLLCQYVA", origChange);
+		AminoAcidChange modChange = AminoAcidChangeNormalizer.shiftSynonymousChange(origChange, "LLLCQYCLLL", "LLLCQYVA");
 				Assert.assertEquals(new AminoAcidChange(6, "CLL", ""), modChange);
 	}
 
 	@Test
-	public void testShiftFrameshiftInsAA() {
+	public void testShiftDeletionFrameshiftInsAA() {
 		AminoAcidChange origChange = new AminoAcidChange(3, "CQY", "C");
-		AminoAcidChange modChange = AminoAcidChangeNormalizer.normalizeDeletion("LLLCQYCLLL", "LLLCQYVA", origChange);
-		Assert.assertEquals(new AminoAcidChange(6, "CLL", "V"), modChange);
+		AminoAcidChange modChange = AminoAcidChangeNormalizer.shiftSynonymousChange(origChange, "LLLCQYCLLL", "LLLCQYVA");
+		Assert.assertEquals(new AminoAcidChange(6, "CL", ""), modChange);
+	}
+
+	@Test
```

Comment:
Added these tests as `AminoAcidChangeNormalizer.shiftSynonymousChange` is now also used for insertions and block substitutions.